request.el

```diff
@@ -441,47 +441,47 @@ PARSER function takes no argument and it is executed in the
 buffer with HTTP response body.  The current position in the HTTP
 response buffer is at the beginning of the buffer.  As the HTTP
 header is stripped off, the cursor is actually at the beginning
-of the response body.  So, for example, you can pass `json-read'
+of the response body.  So, for example, you can pass `json-read\\='
 to parse JSON object in the buffer.  To fetch whole response as a
-string, pass `buffer-string'.
+string, pass `buffer-string\\='.
 
-When using `json-read', it is useful to know that the returned
-type can be modified by `json-object-type', `json-array-type',
-`json-key-type', `json-false' and `json-null'.  See docstring of
+When using `json-read\\=', it is useful to know that the returned
+type can be modified by `json-object-type\\=', `json-array-type\\=',
+`json-key-type\\=', `json-false\\=' and `json-null\\='.  See docstring of
 each function for what it does.  For example, to convert JSON
-objects to plist instead of alist, wrap `json-read' by `lambda'
+objects to plist instead of alist, wrap `json-read\\=' by `lambda\\='
 like this.::
 
     (request
      \"https://...\"
      :parser (lambda ()
-               (let ((json-object-type 'plist))
+               (let ((json-object-type \\='plist))
```

Comment:
This is the only wrong usage of an unescaped single quote.

All the other changes in this patch should be reverted, because they break `text-quoting-style`.